src/ResourceManager/TrafficManager/Commands.TrafficManager2/Profile/NewAzureTrafficManagerProfile.cs

```diff
@@ -52,19 +54,37 @@ public class NewAzureTrafficManagerProfile : TrafficManagerBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string TrafficRoutingMethod { get; set; }
 
+        [Alias("ProtocolForMonitor")]
         [Parameter(Mandatory = true, HelpMessage = "The protocol of the monitor.")]
-        [ValidateSet(Constants.HTTP, Constants.HTTPS, IgnoreCase = false)]
+        [ValidateSet(Constants.HTTP, Constants.HTTPS, Constants.TCP, IgnoreCase = false)]
         [ValidateNotNullOrEmpty]
         public string MonitorProtocol { get; set; }
 
+        [Alias("PortForMonitor")]
```

Comment:
Because I don't want to encourage a mental model where Path, Port and Protocol are peers of RelativeDnsName, TrafficRoutingMethod and Ttl, which would imply that "a Profile has a DNS name, a RoutingMethod, a Port and a Protocol". I want to encourage the mental model: "A Profile has a DNS name, a RoutingMethod and a Monitor. Then, *the Monitor* has a Port and a Protocol".